Makefile.pre.in

```diff
@@ -1981,7 +1981,7 @@ install: @FRAMEWORKINSTALLFIRST@ commoninstall bininstall maninstall @FRAMEWORKI
 			install|*) ensurepip="" ;; \
 		esac; \
 		$(RUNSHARED) $(PYTHON_FOR_BUILD) -m ensurepip \
-			$$ensurepip --root=$(DESTDIR)/ ; \
+			$$ensurepip --root=$(DESTDIR)/ --prefix=$(prefix) ; \
```

Comment:
I'm not sure if pip supports supplying both the `--root` and `--prefix` options together in the same invocation. The tests mock out the actual call to pip, so has anyone actially verified that this works? And even if it does, I doubt there's a guarantee that pip will never make the 2 options mutually incompatible, so it would be better to only ever supply one.